src/styles/app/17-main-tabs.css

```
@@ -64,46 +65,210 @@
+.mainTabsBreadcrumbSepButton:focus-visible,
+.mainTabsBreadcrumbSepButton[data-state="open"] {
+	background: transparent;
+	border-color: transparent;
+	color: var(--text-primary);
+	outline: none;
+}
+
+.mainTabsBreadcrumbSepButton:hover .mainTabsBreadcrumbSep,
+.mainTabsBreadcrumbSepButton:focus-visible .mainTabsBreadcrumbSep,
+.mainTabsBreadcrumbSepButton[data-state="open"] .mainTabsBreadcrumbSep {
+	color: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)